ci/vars-logging-test.yml

```diff
@@ -0,0 +1,7 @@
+---
+# temp: skip os-net-setup
+cifmw_os_net_setup_config: []
```

Comment:
This is not needed, it was added in autoscaling tests because we create some required resources and the networks created in the os_net_setup conflict with the resources we create.
 
```suggestion
```